scripting/retakes.sp

```diff
@@ -463,7 +463,7 @@ public Action Event_PlayerConnectFull(Event event, const char[] name, bool dontB
         return;
     }
 
-    int client = GetClientOfUserId(event.GetInt("userid"));
+    int client = GetClientOfUserId(GetEventInt(event, "userid"));
     SetEntPropFloat(client, Prop_Send, "m_fForceTeam", 3600.0);
```

Comment:
I'm not sure what the purpose of changing these is. As far as I can tell, the only actual change here is the Event_PlayerTeam behavior, right? GetEventInt vs event.GetInt should compile to the same thing.

Reply:
It should be the same, but I'm guessing it is actually different because like I said, when making only this change and disabling other plugins (in my testing I had to disable SMAC) the join team bug does not occur. Also like I said, I think there is something else going on either logically or maybe some handle that was left open, but I don't have the patience to go through it line by line for this somewhat trivial bug.

Again I made only this change and the bug is gone, please try it for yourself.

Reply:
> It should be the same, but I'm guessing it is actually different because like I said, when making only this change and disabling other plugins (in my testing I had to disable SMAC) the join team bug does not occur

I'm willing to merge the 1st change in Event_PlayerConnectFull, which is an *actual change*, but the rest of the lines are no-op changes.  `event.GetInt` and `GetEventInt` compile to the same thing: https://github.com/alliedmodders/sourcemod/blob/b88e0d7875d31bff2f7e337418278c349b783bfd/core/smn_events.cpp#L458-L490. If that's an actual change needed, there's something seriously wrong in sourcemod, which I doubt.